### DISCUSSION TITLE
Offline spend

### DISCUSSION THREAD

**Discussion by yanhong-zhao-ef** [google/lightweight_mmm#34] [General]
For a channel (like offline channel) where I only have the cost data, how should I add it to lightweight mmm? Is the approach to add cost  to media data and the total cost so the cost per unit is just 1?

Thank you!

**Comment by pabloduque0** [comment]
Hello @yanhong-zhao-ef !

For offline channels where only spend data is available, use the spend data as media data. For cost you will use cost as well, so the true cost (just summing your media data for that channel).

Then for media optimization the price you will need to use is 1.
